presto-main/src/main/java/com/facebook/presto/sql/planner/plan/InternalPlanVisitor.java

```diff
@@ -181,4 +181,9 @@ public R visitStatsEquivalentPlanNodeWithLimit(StatsEquivalentPlanNodeWithLimit
     {
         return visitPlan(node, context);
     }
+
+    public R visitNativeExecution(NativeExecutionNode node, C context)
```

Comment:
add override

Reply:
IIRC InternalPlanVisitor.java is the base class for all Internal plan node visit calls so no need to have the override.